core/lib/data/network/config/endpoint.dart

```diff
@@ -2,4 +2,5 @@ import 'package:core/data/network/config/service_path.dart';
 
 class Endpoint {
   static final ServicePath authentication = ServicePath('/api/login');
+  static final ServicePath session = ServicePath('/jmap/session');
```

Comment:
Pro-tip! Prefer `.well-known/jmap`

Reply:
thanks for your notice

Reply:
It allows auto discovery and redirects to jmap/session 

See https://jmap.io/spec-core.html#service-autodiscovery https://datatracker.ietf.org/doc/rfc8615/